src/ci/docker/host-x86_64/x86_64-gnu-llvm-15/script.sh

```diff
@@ -4,7 +4,7 @@ set -ex
 
 # Only run the stage 1 tests on merges, not on PR CI jobs.
 if [[ -z "${PR_CI_JOB}" ]]; then
-    ../x.py --stage 1 test --skip src/tools/tidy
+    ../x.py --stage 1 test --skip src/tools/tidy --skip tests/codegen
```

Comment:
fyi: This script used for llvm16 and llvm17 too.

Reply:
As discussed with @onur-ozkan, I'll set an environment variable to only run gcc tests in CI.

Reply:
I think we should use `SKIP_CODEGEN_TESTS` condition here too

Reply:
Yes that's the only place where I changed it in fact. ;)